operator/v1/types_ingress.go

```diff
@@ -556,6 +572,11 @@ const (
 	AWSNetworkLoadBalancer AWSLoadBalancerType = "NLB"
 )
 
+// AWSSubnetReference is a reference to an AWS subnet. It can be either
+// be a subnet ID or name.
+// +kubebuilder:validation:MaxLength=256
+type AWSSubnetReference string
```

Comment:
The subnet name is a `tag` in AWS.  FWIW I found a reference on tags in https://docs.aws.amazon.com/mediaconnect/latest/ug/tagging-restrictions.html, but I couldn't find anything similar for VPC.